sysbox-in-docker/Makefile

```diff
@@ -21,12 +17,29 @@ SIND_IMAGE := nestybox/sysbox-in-docker
 	ubuntu-bionic   \
 	ubuntu-focal
 
+# Host kernel info
+KERNEL_REL := $(shell uname -r)
+export KERNEL_REL
+
+# Sysbox image-generation globals utilized during the sysbox's building and testing process.
+IMAGE_BASE_DISTRO := $(shell lsb_release -is | tr '[:upper:]' '[:lower:]')
```

Comment:
I purposely avoided this code to offer the user full flexibility to build whichever image they are interested on, not necessarily the one matching their current host distro.